GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -73,7 +73,7 @@ import GTMSessionFetcher
   ///   from `Data` failed.
   /// - Note: See `GTMKeychain.swift` for ``GTMAppAuthSwift/GTMKeychainError``s indirectly thrown by
   ///   calls to this method.
-  @available(macOS 10.15, *)
+  @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
```

Comment:
Why change availability here?  `useDataProtectionKeychain` is a noop for platforms other than macOS, which started supporting it with 10.15.

Reply:
Updated to use `if #available` within the implementation.